pkgs/top-level/all-packages.nix

```diff
@@ -38874,4 +38875,6 @@ with pkgs;
   dillo = callPackage ../by-name/di/dillo/package.nix {
     fltk = fltk13;
   };
+
+  dotultimate = recurseIntoAttrs (callPackages ../development/tools/misc/dotultimate {});
```

Comment:
I think these should go in pkgs/by-name

Reply:
[`pkgs/by-name/README.md`](https://github.com/NixOS/nixpkgs/blob/799661e93525c7fa2c9acf645cafe82bab3ee261/pkgs/by-name/README.md#limitations) seems to still suggest using the category hierarchy for package sets.

Reply:
Hmm... it isn't really a "package set". Just some related packages.

Reply:
You would need to add a dotultimate package, maybe just a symlink join of the two tools if that makes sense here. Then you can add the other packages in the same directory and reference them in passthru.

Users could choose between dotultimate, dotultimate.dotmemory, dotultimate.dottrace then.